src/acvp_aes.c

```diff
@@ -810,6 +810,11 @@ ACVP_RESULT acvp_aes_kat_handler(ACVP_CTX *ctx, JSON_Object *obj) {
         }
 
         ptlen = (unsigned int)json_object_get_number(groupobj, "payloadLen");
+        if(alg_id == ACVP_AES_GMAC && ptlen != 0) {
```

Comment:
nit. Your if statements should always have a space between if and (
There are few more cases in the diff.